native/cocos/platform/android/java/vendor/com/cocos/billing/CocosBillingHelper.java

```diff
@@ -0,0 +1,214 @@
+/****************************************************************************
+Copyright (c) 2024 Xiamen Yaji Software Co., Ltd.
```

Comment:
Should put in `vendor/google`.

Reply:
And it is better to use `GooglePlayBillingHelper.java`. Should check other files too.

Reply:
I named it as something like GoogleBillingXXX, billing is under google not play